index.bs

```diff
@@ -1499,7 +1500,10 @@ input parameters:
 - A list of credentials acceptable to the [=[RP]=] (possibly filtered by the client).
 - Extension data created by the client based on the extensions requested by the [=[RP]=].
 
-When this method is invoked, the [=authenticator=] must perform the following procedure:
+The [=authenticator=] must perform the following procedure depending on the PublicKeyCredentialType of the related [=credential=]:
```

Comment:
Not sure I understand what you are aiming for. 
Looks like you want the first 4 steps under "public-key" to be moved before the "switch" based on on the credential type.  In which case the credential and its type are known.